include/core/parameters.h

```diff
@@ -937,13 +938,16 @@ namespace Parameters
     // ILU or ILUT relative tolerance
     double ilu_precond_rtol;
 
-    // ILU or ILUT fill
+    // AMG parameters either as linear solver preconditioner or as
+    // preconditioner of a coarse-grid solver for LSMG or GCMG
+
+    // ILU or ILUT fill for smoother and coarsener
```

Comment:
what is a `coarsener`?

Reply:
Smoother when you are going down in the V cycle? I always saw it named coarsener, maybe that's just me though.